### ISSUE TITLE
connection-string: code will be rejected by a future version of Rust

### ISSUE THREAD

**Issue by HuakunShen** [Brendonovich/prisma-client-rust#436]
I am getting this warning: `warning: the following packages contain code that will be rejected by a future version of Rust: connection-string v0.1.13`

This package is used by `prisma-engines` and `prisma-client-rust`

`cargo tree -i connection-string` output:

```
connection-string v0.1.13
├── builtin-psl-connectors v0.1.0 (https://github.com/Brendonovich/prisma-engines?tag=pcr-0.6.10#c4aeef82)
│   └── psl v0.1.0 (https://github.com/Brendonovich/prisma-engines?tag=pcr-0.6.10#c4aeef82)
│       ├── datamodel-renderer v0.1.0 (https://github.com/Brendonovich/prisma-engines?tag=pcr-0.6.10#c4aeef82)
│       │   └── sql-introspection-connector v0.1.0 (https://github.com/Brendonovich/prisma-engines?tag=pcr-0.6.10#c4aeef82)
│       │       └── sql-migration-connector v0.1.0 (https://github.com/Brendonovich/prisma-engines?tag=pcr-0.6.10#c4aeef82)
│       │           └── migration-core v0.1.0 (https://github.com/Brendonovich/prisma-engines?tag=pcr-0.6.10#c4aeef82)
│       │               └── prisma-client-rust v0.6.11 (https://github.com/Brendonovich/prisma-client-rust?tag=0.6.11#3ac68d00)
```
